docs/ops/signals/IRDFT_9.md

```diff
@@ -0,0 +1,210 @@
+# Inverse Discrete complex-to-real Fourier Transformation (IRDFT) {#openvino_docs_ops_signals_IRDFT_9}
+
+**Versioned name**: *IRDFT-9*
+
+**Category**: *Signal processing*
+
+**Short description**: *IRDFT* operation performs the inverse complex-to-real discrete Fourier transformation of input tensor by specified dimensions.
+
+**Attributes**:
+
+    No attributes available.
+
+**Inputs**
+
+*   **1**: `data` - Input tensor of type *T* with data for the IRDFT transformation. Type of elements is any supported floating-point type. The last dimension of the input tensor must be equal to 2, that is the input tensor shape must have the form `[D_0, D_1, ..., D_{N-1}, 2]`, representing the real and imaginary components of complex numbers in `[:, ..., :, 0]` and in `[:, ..., :, 1]` correspondingly. **Required.**
```

Comment:
The type description is presented on line #27, so that's redundant.

Reply:
Fixed.